src/main/index.js

```diff
@@ -1,6 +1,6 @@
 /* globals INCLUDE_RESOURCES_PATH */
 import { app } from "electron";
-const autoUpdater = require("electron-updater"); // eslint-disable-line no-unused-vars
+const { autoUpdater } = require("electron-updater");
```

Comment:
For some reason this fixes the "`autoUpdater.checkForUpdatesAndNotify` is not a function" issue

Reply:
👍